lua/astrocommunity/pack/helm/init.lua

```diff
@@ -1,3 +1,37 @@
+local M = {}
+
+function M.setup()
+  vim.filetype.add {
+    extension = {
+      yaml = M.yaml_filetype,
+      yml = M.yaml_filetype,
+      tmpl = M.tmpl_filetype,
+      tpl = M.tpl_filetype,
+    },
+    filename = {
+      ["Chart.yaml"] = "yaml",
+      ["Chart.lock"] = "yaml",
+    },
+  }
+end
```

Comment:
Why isn't this just moved into the astrocore spec?

Reply:
Sorry, I actually meant to do that.

Reply:
Removed.